### DISCUSSION TITLE
Custom mock callbacks

### DISCUSSION THREAD

**Discussion by jsk95** [ashmind/SourceMock#10] [General]
For `Custom mock callbacks` - is this like `.Callback` in Moq? Can I try implement it?

**Comment by ashmind** [comment]
Yep of course you can try -- I'll try not to do anything major that might conflict.

**Reply by jsk95** [reply]
What would you expect in the arg of the callback func? The arguments of the mocked method?

**Reply by jsk95** [reply]
Just an action is like 5 loc change but supporting unlimited arguments/types might be.. a lot of code

**Reply by ashmind** [reply]
> The arguments of the mocked method?

Yep this.
